Bottom: 285a1a78ef6b87cf0153b59f79d710e81f92cdaa
Top:    fef547bddeebc9eb7906785aa5dc70c834c70c09
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:06 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 85bffb4..37340d4 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2707,7 +2707,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
